docs/source/guide/dynamodb.rst

```diff
@@ -0,0 +1,415 @@
+
+Deleting Item
+-------------
+You can also delete the item using :py:meth:`DynamoDB.Table.delete_item`::
+    
+    table.delete_item(
+        Key={
+            'username': 'janedoe',
+            'last_name': 'Doe'
+        }
+    )
+
+
+Batch Writing
+-------------
+If you are loading a lot of data at a time, you can make use of
+:py:meth:`DyanmoDB.Table.batch_writer` so you can both speed up the process and
+reduce the number of write requests made to the service.
+
+This method returns a handle to a batch writer object that will automatically
+handle buffering and sending items in batches.  In addition, the
+batch writer will also automatically handle any unprocessed items and
+resend them as needed.  All you need to do is call ``put_item`` for any
+items you want to add, and ``delete_item`` for any items you want to delete::
+
+    with table.batch_writer() as batch:
+        batch.put_item(
+            Item={
+                'account_type': 'standard_user',
+                'username': 'johndoe',
+                'first_name': 'John',
+                'last_name': 'Doe',
+                'age': 25,
+                'address': {
+                    'road': '1 Jefferson Street',
+                    'city': 'Los Angeles',
+                    'state': 'CA',
+                    'zipcode': 90001
+                }
+            }
+        )
+        batch.put_item(
+            Item={
+                'account_type': 'super_user',
+                'username': 'janedoering',
+                'first_name': 'Jane',
+                'last_name': 'Doering',
+                'age': 40,
+                'address': {
+                    'road': '2 Washington Avenue',
+                    'city': 'Seattle',
+                    'state': 'WA',
+                    'zipcode': 98109
+                }
+            }
+        )
+        batch.put_item(
+            Item={
+                'account_type': 'standard_user',
+                'username': 'bobsmith',
+                'first_name': 'Bob',
+                'last_name':  'Smith',
+                'age': 18,
+                'address': {
+                    'road': '3 Madison Lane',
+                    'city': 'Louisville',
+                    'state': 'KY',
+                    'zipcode': 40213
+                }
+            }
+        )
+        batch.put_item(
+            Item={
+                'account_type': 'super_user',
+                'username': 'alicedoe',
+                'first_name': 'Alice',
+                'last_name': 'Doe',
+                'age': 27,
+                'address': {
+                    'road': '1 Jefferson Street',
+                    'city': 'Los Angeles',
+                    'state': 'CA',
+                    'zipcode': 90001
+                }
+            }
+        )
+
+The batch writer is even able to handle a very large amount of writes to the
+table.
+
+.. warning::
+    If you are running the code blocks in this guide step by step, it may
```

Comment:
Rather than adding this note, why don't we just lower the number of iterations.  The max batch size is 25, so even something like 100/500 would demonstrate this.